`TestSeekForward` flakiness

## Failure

#### Link(s) to logs

- https://github.com/gravitational/teleport/actions/runs/8805031358/job/24166745823

#### Relevant snippet

```
=== Failed
=== FAIL: lib/player  (0.00s)
-test.shuffle 1713895551658968897
panic: test timed out after 10m0s
running tests:
	TestSeekForward (10m0s)

goroutine 8 [running]:
testing.(*M).startAlarm.func1()
	/opt/go/src/testing/testing.go:2366 +0x265
created by time.goFunc
	/opt/go/src/time/sleep.go:177 +0x45

goroutine 1 [chan receive, 10 minutes]:
testing.(*T).Run(0xc000257d40, {0x3c46b71, 0xf}, 0x3d94540)
	/opt/go/src/testing/testing.go:1750 +0x851
testing.runTests.func1(0xc000257d40)
	/opt/go/src/testing/testing.go:2161 +0x86
testing.tRunner(0xc000257d40, 0xc000c67b00)
	/opt/go/src/testing/testing.go:1689 +0x21f
testing.runTests(0xc000235c68, {0x72607e0, 0x8, 0x8}, {0x2?, 0xc000c67bf0?, 0x72958a0?})
	/opt/go/src/testing/testing.go:2159 +0x8bf
testing.(*M).Run(0xc0007fd860)
	/opt/go/src/testing/testing.go:2027 +0xf18
main.main()
	_testmain.go:97 +0x2e5

goroutine 52 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc00058c580)
	/go/pkg/mod/go.opencensus.io@v0.24.0/stats/view/worker.go:292 +0x1a5
created by go.opencensus.io/stats/view.init.0 in goroutine 1
	/go/pkg/mod/go.opencensus.io@v0.24.0/stats/view/worker.go:34 +0xfd

goroutine 54 [chan receive, 10 minutes]:
github.com/jonboulle/clockwork.(*fakeClock).BlockUntil(0xc000aac120, 0x1)
	/go/pkg/mod/github.com/jonboulle/clockwork@v0.4.0/clockwork.go:227 +0x5a
github.com/gravitational/teleport/lib/player_test.TestSeekForward(0xc00064e1a0)
	/__w/teleport/teleport/lib/player/player_test.go:203 +0x5dd
testing.tRunner(0xc00064e1a0, 0x3d94540)
	/opt/go/src/testing/testing.go:1689 +0x21f
created by testing.(*T).Run in goroutine 1
	/opt/go/src/testing/testing.go:1742 +0x826
FAIL	github.com/gravitational/teleport/lib/player	600.311s

=== FAIL: lib/player TestSeekForward (unknown)
```

--- a/lib/player/player_test.go
+++ b/lib/player/player_test.go
@@ -187,6 +187,7 @@ func TestSeekForward(t *testing.T) {
 
 	// advance playback until right before the last event
 	p.SetPos(9001 * time.Millisecond)
+	clk.BlockUntil(1)
 
 	// advance the clock to unblock the player
 	// (it should now spit out all but the last event in rapid succession)

